tests/redteam/poc_json_stack.py

```
@@ -0,0 +1,34 @@
+Difficulty: Script Kiddie
+
+Note: The fix added kMaxJsonDepth=1024 which limits recursion,
+but 1024 stack frames is still enough to exhaust small-stack environments
+(CI runners, containers with 2MB stack).
+"""
+import json, sys, os
+
+def build_deep_json(depth):
+    """Generate deeply nested JSON that tests the parser's recursion limit."""
+    s = "[" * depth + "]" * depth
+    return s
```

Ruff (0.15.9)

[warning] Unnecessary assignment to `s` before `return` statement

Remove unnecessary assignment

(RET504)